ui/src/client/Clients.tsx

```diff
@@ -110,6 +112,15 @@ interface IRowProps {
     fDelete: VoidFunction;
 }
 
+const copyToClipboard = async (text: string) => {
+    try {
+        await navigator.clipboard.writeText(text);
+        console.info("Copied to clipboard:", text);
```

Comment:
For the notifications, Instead of using `cosole.info` and `console.error` I suggest you to make use of `SnackReporter`  to make it look like the example below: 
![image](https://user-images.githubusercontent.com/79609888/235163779-4c0c82a1-09b0-4261-afc2-8cccac34b31e.png)

Reply:
Thanks for the tip! I'm trying to add an additional action to ClientStore to implement it!

Reply:
You can directly inject the SnackManager into the Clients component by adding it to the inject function:
```
export default inject('clientStore', 'snackManager')(Clients);
```

Reply:
Is it better to do it that way? I have already implemented it with a ClientStore.